CHANGELOG.md

```diff
@@ -29,11 +29,12 @@
   * Minimum Supported Deno Version: `1.39.2`.
   * `deno run --allow-net bin/gamedig.js --type tf2 127.0.0.1`
 * Added code examples.
-* New stable field: `queryPort` - this number indicates what was the port that the query was done on, 0 indicates none if not applicable.
 * Fixed `numplayers` not having a default value.
+* New stable field: `queryPort` - this number indicates what was the port that the query was done on, 0 indicates none if not applicable.
```

Comment:
Why this got moved?

Reply:
Oops... I guess I was just grouping the options together haha.

Reply:
Fixed! :D